forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -5234,7 +5227,7 @@ public final boolean isPermanent() {
     }
 
     public final boolean isSpell() {
-        return isInstant() || isSorcery() || (isAura() && !isInZone((ZoneType.Battlefield)));
+        return isInZone(ZoneType.Stack) || isInstant() || isSorcery() || (isAura() && !isInZone(ZoneType.Battlefield));
```

Comment:
The problem is the ability of _Lightning Storm_
when _Chimil_ is in play you can now no longer _Stifle_ it 💩

Reply:
reworked again